### DISCUSSION TITLE
Layout::modal()->open() doesn't work

### DISCUSSION THREAD

**Discussion by mixanrulezzz** [orchidsoftware/platform#2030] [Q&A]
I tried to open modal without ModalToggle, but Layout::modal()->open(true) doesn't work. Maybe I missed adding something? 

I add script ```$('body').find('.modal[data-modal-open=1]').modal('show');``` into my code and my modal opened, but it strange that I need to add script for that

**Comment by thewebartisan7** [comment]
I confirm this happen also to me. I see that in frontend it's correctly set the data-model-open, but the method open in  modal_controller is never fired:

```
    // I suppose this should fire the open modal when found data-model-open
    open(options) {
       //...
        $(this.element).modal('toggle');
    }
```

So maybe is missing in initialize method the call to open if found data-model-open? 

I am still new with stimulus hotwired, so maybe I am missing something.
